biodatasets/seth_corpus/seth_corpus.py

```diff
@@ -0,0 +1,224 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Descriptions of genetic variations and their effect are widely spread across the biomedical literature. However,
+finding all mentions of a specific variation, or all mentions of variations in a specific gene, is difficult to
+achieve due to the many ways such variations are described. Here, we describe SETH, a tool for the recognition of
+variations from text and their subsequent normalization to dbSNP or UniProt. SETH achieves high precision and recall
+on several evaluation corpora of PubMed abstracts. It is freely available and encompasses stand-alone scripts for
+isolated application and evaluation as well as a thorough documentation for integration into other applications.
+The script loads dataset in bigbio schema (using knowledgebase schema: schemas/kb) AND/OR source (default) schema """
+
+from typing import Dict, List, Tuple
+from pathlib import Path
+
+import datasets
+from utils import parsing, schemas
+from utils.configs import BigBioConfig
+from utils.constants import Tasks
+
+_CITATION = """\
+@Article{SETH2016,
+    Title       = {SETH detects and normalizes genetic variants in text.},
+    Author      = {Thomas, Philippe and Rockt{"{a}}schel, Tim and Hakenberg, J{"{o}}rg and Lichtblau, Yvonne and Leser, Ulf},
+    Journal     = {Bioinformatics},
+    Year        = {2016},
+    Month       = {Jun},
+    Doi         = {10.1093/bioinformatics/btw234},
+    Language    = {eng},
+    Medline-pst = {aheadofprint},
+    Pmid        = {27256315},
+    Url         = {http://dx.doi.org/10.1093/bioinformatics/btw234
+}
+"""
+
+_DATASETNAME = "seth_corpus"
+
+_DESCRIPTION = """SNP named entity recognition corpus consisting of 630 PubMed citations."""
+
+_HOMEPAGE = "https://github.com/rockt/SETH"
+
+_LICENSE = "Apache License 2.0"
+
+_URLS = {
+    "source": "https://github.com/rockt/SETH/archive/refs/heads/master.zip",
+    "bigbio_kb": "https://github.com/rockt/SETH/archive/refs/heads/master.zip",
+}
+
+_SUPPORTED_TASKS = [Tasks.NAMED_ENTITY_RECOGNITION]
```

Comment:
`_SUPPORTED_TASKS` should include `Tasks.RELATION_EXTRACTION`

Reply:
This task was added to the script

Reply:
This task was added in the script.